public/app/features/explore/SecondaryActions.tsx

```diff
@@ -21,6 +21,11 @@ const getStyles = (theme: GrafanaTheme2) => {
     containerMargin: css`
       margin-top: ${theme.spacing(2)};
     `,
+    exploreActiveButton: css`
+      box-shadow: ${theme.shadows.z1};
+      border: 1px solid ${theme.colors.warning.text} !important;
+      color: ${theme.colors.warning.text} !important;
```

Comment:
I think we can use the `theme.colors.warning.border` token for the border, also `!important` here shouldn't be necessary:
```suggestion
      border: 1px solid ${theme.colors.warning.border};
      color: ${theme.colors.warning.text};
```